test/functional/terminal/channel_spec.lua

```diff
@@ -198,4 +198,18 @@ describe('no crash when TermOpen autocommand', function()
     ]]}
     assert_alive()
   end)
+
+  it('convert newline when using nvim_open_term()', function()
+    local buf = meths.create_buf(false, true)
+    local win = meths.get_current_win()
+    local term = meths.open_term(buf, {force_crlf = true})
+    meths.win_set_buf(win, buf)
+    meths.chan_send(term, 'here\nthere')
```

Comment:
also test the result of sending `\r\n`. don't add another it() block, just add another case to this test.

Reply:
test case updated.